components/Tweet/Tweet.js

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import { TwitterShareButton } from "react-twitter-embed"
+
+const Tweet = () => {
+  return (
+    <div className="centerContent">
+      <div className="selfCenter spaceBetween">
+        <TwitterShareButton
+          onLoad={function noRefCheck() {}}
+          options={{
+            text: "I just provided digital testimony on Bill H100 at "
+            // via: "saurabhnemade"
+          }}
+          url="https://digital-testimony-dev.web.app/bill?id=H100"
```

Comment:
Instead of hardcoding `https://digital-testimony-dev.web.app`, we should use `location.origin`, so the url always matches the domain name of the site.

Reply:
I like it.  I am getting a weird weird error though: ReferenceError: location is not defined

![image](https://user-images.githubusercontent.com/67333843/160250857-579af590-8596-4128-8ab1-4f5b4f1ae17d.png)

Reply:
I found window.location.href which seems to work.